ivy/array/elementwise.py

```diff
@@ -1381,6 +1381,27 @@ def log2(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
             an array containing the evaluated base ``2`` logarithm for each element
             in ``self``. The returned array must have a real-valued floating-point
             data type determined by :ref:`type-promotion`.
+
+        Examples
+        --------
+        Using :code:`ivy.Array` instance method:
+
+        x = ivy.array([4.0, 1, -0.0, -5.0])
+        >>> y = x.log2()
+        >>> print(y)
+        ivy.array([2., 0., -inf, nan])
+
+        >>> x = ivy.array([float('nan'), -4.0, -0.0, 1.0, 4.0, float('+inf')])
+        >>> y = x.log2()
+        >>> print(y)
+        ivy.array([nan, nan, -inf, 0., 2., inf])
+
+        >>> x = ivy.array([[float('nan'), 1, 4.0, float('+inf')],\
```

Comment:
For correct rendering, we pass `...` to indicate that it's a new line instead of using brake lines

Example:
Instead of using this:

```
>>> x = ivy.Container(a=ivy.array([0.0, float('nan')]), \
                              b=ivy.array([-0.0, -3.9, float('+inf')]), \
                              c=ivy.array([3.3, 1.1, 1.]))
```
We can adopt to use this:

```
>>> x = ivy.Container(a=ivy.array([0.0, float('nan')]),
...                              b=ivy.array([-0.0, -3.9, float('+inf')]), 
...                              c=ivy.array([3.3, 1.1, 1.]))
```